BaseStation/BaseServer/BaseServer.cs

```diff
@@ -37,33 +40,19 @@ private static void ClientConnected(object sender, EventArgs e)
             Console.WriteLine(Scarlet.Communications.Server.GetClients());
         }
 
-        private static short PreventOverflow(short shortVal)
-        {
-            if (shortVal == -32768)
-            {
-                shortVal++;
-            }
-            return shortVal;
-        }
+        private static long lastControlSend = 0;
 
-        public static void EventLoop()
+        public static void Update(Controller controller)
```

Comment:
This method needs to tell the rover to not move if the controller is not connected. Whether you need to do that explicitly or not needs to be well-tested since we will damage our rover otherwise. The wheel motor controllers will stop by themselves if you send nothing but the Talons on the rack and the arm will not!

Reply:
Whoops looks like the change required for this resides in a different branch, I'll work on merging it into here.